tests/RelationTest.php

```diff
@@ -66,10 +69,10 @@ public function testNoForeignType()
 
         $this->manager = new Manager($type_builder, $relation_builder, $types);
         $this->expectException('Aura\Marshal\Exception');
-        $this->manager->posts;
+        $this->manager->__get('posts');
```

Comment:
Why did you changed this ?

Reply:
I do understand the effect is same, but wished those changes are not made to satisfy phpstan. This is tests file right ?

Reply:
@compwright It's up to @harikt to resolve the conversation when he sees fit.